test/tests/multiapps/loose_couple_time_adapt/tests

```diff
@@ -8,14 +8,14 @@
     [begin]
       type = 'Exodiff'
       input = 'begin.i'
-      exodiff = 'begin.e begin_out_dummy0.e'
+      exodiff = 'begin.e begin_dummy0.e'
```

Comment:
So I'm not sure I like this change. Here you can see that we are using a user override name for the `file_base` (e.g. "begin"). When looking at the SubApp output though, we still have an "out" parameter. That's a little bit odd. Consider what would happen if I specified "begin_out". I'd get "begin_out_out". Contrast that with what happens when you don't specify a default, you get "filename_out" for the master and "<master name>_<subname>_out". It's inconsistent between the two cases.